eq-author/src/App/QuestionnaireDesignPage/index.js

```diff
@@ -215,8 +217,15 @@ export default withMutations(props => (
       },
     }}
   >
-    {innerProps => (
-      <UnwrappedQuestionnaireDesignPage {...innerProps} {...props} />
-    )}
+    {innerProps => {
+      if (!innerProps.data || !innerProps.data.questionnaire) {
```

Comment:
There are some code paths that check for `loading` in `render` that I don't think will ever run now. As we will never have data whilst we are loading.

I presume we still want to show loading messages

Reply:
refactored so <UnwrappedQuestionnaireDesignPage /> always returned unless component not loading when no questionnaire data.